### ISSUE TITLE
Add command that returns used mods

### ISSUE THREAD

**Issue by Ekwav** [Coflnet/SkyblockMod#81]
There should be a command `getMods` that returns which mods are being used.  
To detect and mitigate compatibility issues with mods like Optifine (which changes the font) that information is needed on the server.  
The command should respond with the command `foundMods` with the body 
```
{
"fileNames": ["file names", "in the mods folder"],
"modNames": ["modName", "from forge"],
"fileHashes": ["base64 md5Hashes of mod files", "to detect version"]
}
```

**Comment by Sasuke449** [comment]
I'll do this!

**Comment by Sasuke449** [comment]
fixed in #82
